deploy/helm/airflow-operator/values.yaml

```diff
@@ -41,3 +41,8 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+# When running on a non-default Kubernetes cluster domain and the auto detections is not working correctly,
+# you can set your custom cluster domain here.
+# See the https://docs.stackable.tech/home/stable/guides/kubernetes-cluster-domain guide for details
+# kubernetesClusterDomain: my-cluster.local
```

Comment:
Not sure this makes sense to even mention it in the values, since this wont do anything?
```suggestion

# When running on a non-default Kubernetes cluster domain and the auto detection is not working correctly,
# you can set your custom cluster domain here.
# See the https://docs.stackable.tech/home/stable/guides/kubernetes-cluster-domain guide for details
# kubernetesClusterDomain: my-cluster.local
```

Reply:
Was thinking the same thing, no strong opinion.
It's mostly about if we start reading the field in the operator we *might* forget to document the property. I let you decide